hummingbot/core/network_base.py

```diff
@@ -120,7 +119,7 @@ def check_network_timeout(self, timeout):
                 self.logger().error("Unexpected error starting or stopping network.", exc_info=True)
 
     def start(self):
-        self._check_network_task = safe_ensure_future(self._check_network_loop())
+        self._check_network_task = asyncio.ensure_future(self._check_network_loop())
```

Comment:
Why don't want to use `safe_ensure_future` here?

Reply:
Oh that was a mistake. I was trying to do a diff test to see if `safe_ensure_future()` caused memory leak.

Reply:
I'm going to create another PR to fix that.